portfolio/src/main/java/com/google/sps/servlets/RandomPhraseServlets.java

```diff
@@ -10,7 +10,7 @@
 @WebServlet("/random-phrase")
 public class RandomPhraseServlets extends HttpServlet {
 
-    String[] funFacts = {
+    public static final String[] funFacts = {
```

Comment:
In the future, Java constants are typically stylized in all caps with words separated by underscores, so this would be `FUN_FACTS`. I don't want to block this PR waiting on this change so I'll just mention it!